.gitlab-ci.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+  - test
+
+include:
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-commits.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-license.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-shell-format.yml'
+
+test:check-shell-formatting:
+  before_script:
+    - SHELL_SCRIPTS=/dev/null
+
```

Comment:
this is a placeholder,without it, and as I just found out, shell-format fails in the absence of files matching `*.sh`

Reply:
then will you delete this before script afterwards?

Reply:
sure! you know, I just need to kick start this repo. please :)

Reply:
Alex, please. I need the checks in. if I remove the `SHELL_SCRIPTS` it will not work. I have a queue of PRs. The `SHELL_SCRIPTS=` overwrite will stay, but it will point to the code (the files will not have .sh extension). I need this in. Why are you blocking it?

Reply:
Ok. It seems like it's an urgency and it's no opportunity for me to understand it better. Please follow up and don't pay attention on my comments, mostly it's because a lack of understanding here.

Reply:
> Ok. It seems like it's an urgency and it's no opportunity for me to understand it better. Please follow up and don't pay attention on my comments, mostly it's because a lack of understanding here.

no  worries. thank you. I will explain once I have a spare moment :) we can schedule a call.

Reply:
this is the final version: https://github.com/mendersoftware/app-update-module/blob/master/.gitlab-ci.yml#L14 so as you can see, it greatly helped to have this up front, since there were 3 independent PRs requiring that.